### PR TITLE
#45: Written down the tooltip directive based on the popup directive

### PR DESCRIPTION
**Pull request by atd-schubert**
First vacation directive

### CODE REVIEW

File: Leaflet2Angular.md [hunk 1]
```diff
@@ -407,6 +407,9 @@ Directive name in Angular2: `yaga-tooltip`.
 * ng-content(HTML)
 * `[(opened)]: boolean`
 * `[(opacity)]: number`
+* `[(lat)]: number`
+* `[(lng)]: number`
+* `[(position)]: LatLng`
```
Comment (atd-schubert) [inline Leaflet2Angular.md:412]: These were missing in the documentation...

File: ts/index.ts [hunk 2]
```diff
@@ -3,15 +3,19 @@ export * from './map.component';
 export * from './tile-layer.directive';
 export * from './image-overlay.directive';
 export * from './popup.directive';
+export * from './tooltip.directive';
```
Comment (OpenSteveMap) [inline ts/index.ts:8]: conflict with icon directive and div icon directive

**Review by OpenSteveMap — COMMENTED** [review]
merge conflict cause of merged icon directive

**Review by OpenSteveMap — COMMENTED** [review]
can me merged now

**Review by OpenSteveMap — APPROVED** [review]
ok

**Pull request merged**